### ISSUE TITLE
getComputedStyle is not defined in SSR for ng-primitives tooltip

### ISSUE THREAD

**Issue by benjaminforras** [ng-primitives/ng-primitives#83]
### Affected Primitive

Tooltip

### Version

0.14.0

### Bug Description

When running the development server with SSR enabled, the application crashes due to `getComputedStyle` not being defined in the server environment. This occurs in the `NgpTooltip` component's `ngOnInit` lifecycle hook.

## Environment
- Angular CLI: 19.0.2
- Node.js: v20.18.0
- ng-primitives: v0.14.0

### Steps to Reproduce

Add the following to an element:
```html
    [ngpTooltipTrigger]="template"
    [ngpTooltipTriggerPlacement]="'right'"
    [ngpTooltipTriggerOffset]="12"
    [ngpTooltipTriggerDisabled]="false"
    [ngpTooltipTriggerOpen]="true
```

**Comment by ashley-hunter** [comment]
Thanks! Should be fixed in 0.15.

**Comment by benjaminforras** [comment]
That was fast! Thank you!
